### PR TITLE
Added contact page

### PR DESCRIPTION
**Pull request by pritishposwal**
Added contact page along with it's components for address and map

### CODE REVIEW

File: app/contact/page.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import ContactInfo from '@/components/contact/ContactInfo';
+import AddressInfo from '@/components/contact/AddressInfo';
+import LocationMap from '@/components/contact/LocationMap';
+
+export default function ContactPage() {
+  return (
+    <div className="container mx-auto mb-0">
+      <h1 className="text-4xl font-body font-bold text-center mt-12 mb-10 text-primary-dark">
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply app/contact/page.js]: fixed in latest commit to my branch

File: app/contact/page.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import ContactInfo from '@/components/contact/ContactInfo';
+import AddressInfo from '@/components/contact/AddressInfo';
+import LocationMap from '@/components/contact/LocationMap';
+
+export default function ContactPage() {
+  return (
+    <div className="container mx-auto mb-0">
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply app/contact/page.js]: fixed in latest commit to my branch

[the remark this answers is not in the document]
Comment (pritishposwal) [reply app/contact/page.js]: fixed in latest commit to my branch

File: components/contact/AddressInfo.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+function AddressInfo({ address }) {
+  const addressLines = address.split(',,');
+  return (
+    <div className="p-4 font-body text-primary-dark">
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply components/contact/AddressInfo.js]: fixed in latest commit to my branch

File: components/contact/ContactInfo.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+// components/contact/ContactInfo.js
+function ContactInfo({ phone, email, phonedept, emaildept, fax }) {
+  return (
+    <div className="p-4 font-body text-primary-dark">
+      <h3 className="text-3xl font-bold">Contact</h3>
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply components/contact/ContactInfo.js]: fixed in latest commit to my branch

File: components/contact/AddressInfo.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (pritishposwal) [reply components/contact/AddressInfo.js]: fixed in latest commit to my branch

File: components/contact/AddressInfo.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+      <h3 className="text-3xl font-bold">Address</h3>
+      {addressLines.map((line, index) =>
+        index === 0 ? (
+          <p key={index} className="text-xl mt-2 font-bold">
+            {line}
+          </p>
+        ) : (
+          <p key={index} className="text-xl ">
+            {line}
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply components/contact/AddressInfo.js]: fixed in latest commit to my branch

File: components/contact/AddressInfo.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+function AddressInfo({ address }) {
+  const addressLines = address.split(',,');
+  return (
+    <div className="p-4 font-body text-primary-dark">
+      <h3 className="text-3xl font-bold">Address</h3>
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply components/contact/AddressInfo.js]: fixed in latest commit to my branch

File: components/contact/AddressInfo.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+      {addressLines.map((line, index) =>
+        index === 0 ? (
+          <p key={index} className="text-xl mt-2 font-bold">
```
[the remark this answers is not in the document]
Comment (pritishposwal) [reply components/contact/AddressInfo.js]: fixed in latest commit to my branch

**Comment by aryanteng** [comment]
Screenshot 2024-03-07 at 7 00 41 PM
this is, again, not responsive, please do not push any code that is not responsive, you should first ensure your page works in all screen widths and then push the code, also the text below the contact and address should be left-aligned not centre-aligned

**Review by aryanteng — APPROVED** [review]

**Pull request merged**